### PR TITLE
parcel v2.3 support

### PR DESCRIPTION
**Pull request by bycaldr**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export default new Resolver({
 		const result = attemptResolve(filePath, pathsMap, logger);
 		logger.verbose({ message: `Result: ${result}` });
 		return {
-			filePath: result,
+			filePath: path.join(__dirname, '../../../../', result),
```
Comment (bycaldr) [inline src/index.ts:24]: Yep. Not the best solution, but I haven't found a different way to get the project root or project package.json location to resolve the absolute path.

Comment (cangSDARM) [reply src/index.ts:24]: I got this error:

```
parcel-resolver-tspaths: The "path" argument must be of type string. Received null
  at Object.resolve (C:\Users\Allen\Desktop\Personal\tauri-parcel-react\node_modules\parcel-resolver-tspaths\dist\min\index.min.js:1:2965)
```

my config:
```
// tsconfig:
"baseUrl": ".",
"paths": {
    "@/*": ["client/src/*"],
    "~/*": ["./*"]
 },

// load:
import Layout from "@/components/Layout";
```

**Comment by cangSDARM** [comment]
I got this error:

```
parcel-resolver-tspaths: The "path" argument must be of type string. Received null
  at Object.resolve (\node_modules\parcel-resolver-tspaths\dist\min\index.min.js:1:2965)
```

my config:
```
// tsconfig:
"baseUrl": ".",
"paths": {
    "@/*": ["client/src/*"],
    "~/*": ["./*"]
 },

// load:
import Layout from "@/components/Layout";
```

File: src/index.ts (hunk 1 above)
Comment (cangSDARM) [reply src/index.ts:24]: maybe u can use something like this:

```ts
const appDirectory = fs.realpathSync(process.cwd());

const resolveApp = (...relativePaths) => path.resolve(appDirectory, ...relativePaths);
```

Comment (zachbryant) [reply src/index.ts:24]: Seems like parcel changed the requirements and started to enforce it. Fixed in the coming version.

For reference, this was the offending line:
`return path.relative('.', absolutePath); // parcel expects path from the project root`

**Comment by zachbryant** [comment]
Closing - fixed in latest commits. Thanks for identifying the issues though!

**Pull request closed without merging**